src/components/main-view/main-view.jsx

```diff
@@ -32,7 +35,7 @@ export const MainView = () => {
           Title: movie.Title,
           Genre: movie.Genre[0].name,
           Description: movie.Description,
-          Director: movie.Director.Name
```

Comment:
Hi @chrisdvadashanta ,

It seems that a change has been made, where the `Director` is now represented as an object with multiple properties. Consequently, it will be necessary for you to make updates in your code wherever you previously utilized the Director as a string.